src/screens/Contact.js

```diff
@@ -22,8 +23,13 @@ class Contact extends Component {
 
   componentDidMount() {
     getContacts((contacts) => {
-      console.log(contacts);
-      this.setState({ contacts })
+
+      sha512Hash(contacts[0].recordID, (hash) => {
+        console.log(contacts[0].recordID);
+        console.log(hash);
+      });
```

Comment:
This is working well. I just have to now build the networking layer to send it to the backend